src/main/java/core/basesyntax/Storage.java

```diff
@@ -0,0 +1,50 @@
+package core.basesyntax;
+
+import java.time.LocalDate;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    public static Map<String, DateAndQuantityPair> storage = new HashMap<>();
+
+    public static class DateAndQuantityPair {
+        private LocalDate date;
+        private Integer quantity;
+        private DateAndQuantityPair next = null;
```

Comment:
```suggestion
        private DateAndQuantityPair next;
```
null will be assigned by default, you don't need to do it explicitly